docs_src/design/TOC.md

```diff
@@ -4,6 +4,8 @@
 | Name/Link                                                                 | Short Description                                             |
 |---------------------------------------------------------------------------|---------------------------------------------------------------|
 | [0001 System Events for Devices](./ucr/0001-System-Events-for-Devices.md) | Use Case for System Events for Device add/update/delete |
+| [0002 Device Parent-Child Relationships](./ucr/0002-Device-Parent-Child-Relationships.md) | Use Case for Device Parent-Child Relationships |
+| [0003 Extending Device Data](./ucr/0003-Extending-Device-Data.md) | Use Case for Application Services Extending Device Data |
```

Comment:
I think the two UCRs are not so related. Maybe from an implementation perspective, as author mentions:
> Potentially related to Use Case for Device Parent-Child Relationships [0002 Device Parent-Child Relationships](./0002-Device-Parent-Child-Relationships.md), if the "alternative path" solution is chosen here.

I suggest separating the PR because this would also make the TCS approval complicated. Will there be two votes or one? Is there a way to know if both UCRs were peer reviewed sufficiently, especially since one was added later?

Reply:
OK, good points; I'll separate into two UCRs.